src/components/slider/slider.js

```diff
@@ -254,6 +353,29 @@ function SliderDirective($$rAF, $window, $mdAria, $mdUtil, $mdConstant, $mdThemi
       }
     }
 
+    function mouseDownListener() {
+      redrawTicks();
+
+      scope.mouseActive = true;
+      wrapper.removeClass('md-focused');
+
+      $timeout(function() {
+        scope.mouseActive = false;
+      }, 100);
```

Comment:
After reading @marcysutton post, i think it's necessary although it seems to be working with `$mdUtil.nextTick()`